api/utils.ts

```diff
@@ -49,6 +50,28 @@ export function coerceCredentials(warehouse: string, credentials: any): Credenti
   }
 }
 
+export async function testCredentials(credentials: Credentials, warehouse: string) {
```

Comment:
if we are going to call this via the api should it not be in it's own command file like the other commands? Seems strange for us to import this from `@dataform/api/utils`

Reply:
Either way, you want to export this in the api/index.ts file probably so it can just be used directly from importing `@dataform/api`

Reply:
As discussed, moved all credentials related stuff to `credentials.ts` and exported that.